final release/format_data.py

```diff
@@ -0,0 +1,208 @@
+
+# coding: utf-8
+
+# In[45]:
```

Comment:
Remove these comments.

Reply:
these should already be removed. I must have used the older version. Not a big deal almost exactly the same

Reply:
Right, there were two commits in this PR. I'll check the other one.

Reply:
ok